prow/prowjobs/kubeflow/kubeflow-periodics.yaml

```diff
@@ -0,0 +1,186 @@
+periodics:
+- name: kubeflow-periodic-0-7
+  cluster: kubeflow
+  interval: 8h
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+    - image: gcr.io/kubeflow-ci/test-worker:latest
+      imagePullPolicy: Always
+      env:
+      - name: REPO_OWNER
+        value: kubeflow
+      - name: REPO_NAME
+        value: kubeflow
```

Comment:
This should be kfctl

Reply:
@jlewi I just copied from here: https://github.com/kubernetes/test-infra/blob/0326e2563add4824108d877e5b2c9ce3eee594f2/config/jobs/kubeflow/kubeflow-periodics.yaml#L13

So I want to confirm, line 15 (the value for `REPO_NAME`) should be "kfctl" instead of "kubeflow"?

Reply:
Yes it should be kfctl. It was changed in a subsequent PR.
https://github.com/kubernetes/test-infra/blob/0d180418583d65518a976b339219861ac5fafb5b/config/jobs/kubeflow/kubeflow-periodics.yaml#L16

Reply:
Fixed.